src/types/indexeddb.js

```diff
@@ -133,15 +133,15 @@ export interface IDBIndex<K, L, V> extends EventTarget {
 }
 
 export interface IDBKeyRange<K> {
-  static bound<J>(
+  bound<J>(
```

Comment:
PLEASE VERIFY! I'm not really sure here.

Reply:
Me neither.
I believe Flow removed the "static" properties on interface recently and that's why you need to do this, right?

One issue is that window.IDBKeyRange is both a factory (with only/bound/lowerBound/upperBound static functions) and the class type for instances returned by these functions. I'm not sure how to type this, I tried with a `declare class` but it didn't work either.

But I don't think this interface is really taken into account by flow at the moment anyway... This is something else that broke when I upgraded to v0.54. Everything looks like as if window.IDBKeyRange is any...

But I know that flow-typed/overrides/Window.js is still taken into account because if I try to call something wrong on `geckoProfiler` Flow catches it properly.

Well, could you file a bug, and put a // TODO with the bug# here and/or in flow-typed/overrides/Window.js ?